README.md

```diff
@@ -225,3 +225,12 @@ You can define environment variables for dev-mode in .env.development file withi
 npm ci
 npm run dev
 ```
+
+## Release flow
+
+When you receive the approval of your pull-request from the code owners and pass all the checks, please do the following:
```

Comment:
I think we could add here some info about commit message format and it's relation with publishing version